From: Hilko Bengen <bengen@debian.org>
Date: Sun, 10 Jan 2021 18:21:00 +0100
Subject: Disable some tests

These have failed on buildds but failures could not be reproduced in
regular schroot/sbuild environments.
---
 tests/Makefile.am | 31 -------------------------------
 1 file changed, 31 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index e108768..0101d25 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -329,37 +329,6 @@ test_socket_activation_CPPFLAGS = \
 test_socket_activation_CFLAGS = $(WARNINGS_CFLAGS)
 endif
 
-if !IS_WINDOWS
-TESTS += test-stdio.sh
-# check_LTLIBRARIES won't build a shared library (see automake manual).
-# So we have to do this and add a dependency.
-noinst_LTLIBRARIES += \
-	test-stdio-plugin.la \
-	$(NULL)
-test-stdio.sh: test-stdio-plugin.la
-
-test_stdio_plugin_la_SOURCES = \
-	test-stdio-plugin.c \
-	$(top_srcdir)/include/nbdkit-plugin.h \
-	$(NULL)
-test_stdio_plugin_la_CPPFLAGS = \
-	-I$(top_srcdir)/include \
-	-I$(top_builddir)/include \
-	-I$(top_srcdir)/common/replacements \
-	$(NULL)
-test_stdio_plugin_la_CFLAGS = $(WARNINGS_CFLAGS)
-# For use of the -rpath option, see:
-# https://lists.gnu.org/archive/html/libtool/2007-07/msg00067.html
-test_stdio_plugin_la_LDFLAGS = \
-	-module -avoid-version -shared $(NO_UNDEFINED_ON_WINDOWS) -rpath /nowhere \
-	$(NULL)
-test_stdio_plugin_la_LIBADD = \
-	$(top_builddir)/common/replacements/libcompat.la \
-	$(top_builddir)/common/utils/libutils.la \
-	$(IMPORT_LIBRARY_ON_WINDOWS) \
-	$(NULL)
-endif
-
 # check_LTLIBRARIES won't build a shared library (see automake manual).
 # So we have to do this and add a dependency.
 noinst_LTLIBRARIES += \
